ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.c

```diff
@@ -100,7 +100,9 @@ tflite_loadModelFile (const GstTensorFilterProperties * prop,
 static int
 tflite_open (const GstTensorFilterProperties * prop, void **private_data)
 {
-  return tflite_loadModelFile (prop, private_data);
+  int ret = tflite_loadModelFile (prop, private_data);
+  g_assert (ret == 0);       /** This must be called only once */
+  return 0;
```

Comment:
It's ok to add assert; however, let's keep the return value (return ret) as we may "ignore" g_assert at deployment in some cases.

Reply:
I'll apply it